src/sentry/api/endpoints/organization_code_mappings.py

```diff
@@ -133,15 +133,18 @@ def get(self, request: Request, organization) -> Response:
 
         integration_id = request.GET.get("integrationId")
         project_id = request.GET.get("projectId")
+        all_items = request.GET.get("allItems")
 
-        if not integration_id and not project_id:
+        if not (integration_id or project_id or all_items):
             return self.respond(
-                {"detail": 'Missing valid "projectId" or "integrationId"'},
+                {"detail": 'Missing valid "projectId" or "integrationId" or "allItems"'},
                 status=status.HTTP_400_BAD_REQUEST,
             )
 
         queryset = RepositoryProjectPathConfig.objects.all()
 
+        if all_items:
+            queryset = queryset.filter(project__in=self.get_projects(request, organization))
```

Comment:
In other endpoints we use `-1` to mean all projects the current user is a member of and `''` as 'all projects'. Should we use the same conventions here as well? It would let you simplify the condition on 153-156 as well.